### PR TITLE
fix(mobile): clarify separate tokens

### PR DESCRIPTION
**Pull request by ghost**
Jira: 89540

### CODE REVIEW

File: src/content/docs/mobile-monitoring/new-relic-mobile-flutter/monitor-your-flutter-application.mdx [hunk 1]
```diff
@@ -110,7 +110,7 @@ import 'package:newrelic_mobile/newrelic_mobile.dart';
     class MyApp extends StatelessWidget {
     ....
 ```
-Make sure you pasted your application token(s) (copied from [Step 3](#app-token)) into `appToken = ""` in the code above. If you're monitoring both Android-native and iOS-native apps, you'll need to add separate tokens for both.
+Make sure you pasted your application token(s) (copied from [Step 3](#app-token)) into `appToken = ""` in the code above. If you deployed your hybrid app to both iOS and Android, you'll need to add two separate tokens, one for iOS and one for Android.
```
Comment (jeff-colucci) [inline src/content/docs/mobile-monitoring/new-relic-mobile-flutter/monitor-your-flutter-application.mdx:113]: I think there should be a colon after "two separate tokens" here.

File: src/content/docs/mobile-monitoring/new-relic-mobile-maui-dotnet/monitor-your-net-maui-application.mdx [hunk 2]
```diff
@@ -89,7 +90,8 @@ using NewRelic.MAUI.Plugin;
       }
     }
 ```
-2. In the config file above, replace `<YOUR_APP_TOKEN>` with your custom application token(s) (copied from [Step 3]. If you're monitoring both .NET Android and .NET iOS apps, you'll need to add two different application tokens.
+
+Make sure you pasted your application token(s) into `appToken = ""` in the code above. If you deployed your hybrid app to both iOS and Android, you'll need to add two separate tokens, one for iOS and one for Android.
```
Comment (jeff-colucci) [inline src/content/docs/mobile-monitoring/new-relic-mobile-maui-dotnet/monitor-your-net-maui-application.mdx:94]: Again, I think a colon is correct here.

File: src/content/docs/mobile-monitoring/new-relic-mobile-xamarin/monitor-your-xamarin-application.mdx [hunk 3]
```diff
@@ -92,7 +93,8 @@ using NewRelic.Xamarin.Plugin;
       }
     }
 ```
-2. In the config file above, replace `<YOUR_APP_TOKEN>` with your custom application token(s) (copied from [Step 3]. If you're monitoring both Xamarin Android and Xamarin iOS apps, you'll need to add two different application tokens.
+
+Make sure you pasted your application token(s) into `appToken = ""` in the code above. If you deployed your hybrid app to both iOS and Android, you'll need to add two separate tokens, one for iOS and one for Android.
```
Comment (jeff-colucci) [inline src/content/docs/mobile-monitoring/new-relic-mobile-xamarin/monitor-your-xamarin-application.mdx:97]: Colon once more

File: src/content/docs/mobile-monitoring/new-relic-mobile/maintenance/configure-settings-for-mobile-monitoring.mdx [hunk 4]
```diff
@@ -112,16 +112,15 @@ To configure these settings, go to **[one.newrelic.com > All capabilities](https
 
 ## View your application token [#application-token]
 
-One of our [data ingest keys](/docs/apis/intro-apis/new-relic-api-keys) is our mobile monitoring application token. For each mobile app you monitor, we automatically assign a unique, 40-character hexadecimal string to that app, for both the US and EU datacenter. This token is required to authenticate your app's data.
+Your mobile app token is a unique, 40-character hexadecimal string that authenticates your app's data to New Relic. Each mobile app you monitor has a unique token for both the US and EU datacenters.
+
+All data from your mobile app is reported to the same place in the New Relic mobile monitoring UI, regardless of the app's name when installed on a mobile device. If you're using a hybrid framework such as Flutter or React Native and deploying to both iOS and Android platforms, you'll need to use separate tokens, one for iOS and one for Android.
```
Comment (jeff-colucci) [inline src/content/docs/mobile-monitoring/new-relic-mobile/maintenance/configure-settings-for-mobile-monitoring.mdx:117]: Colon, I think the second sentence should be broken up more. Also in the first sentence you can change "is reported" to "reports" for a quick passive voice fix, I think.

File: src/content/docs/mobile-monitoring/new-relic-monitoring-react-native/monitor-your-react-native-application.mdx [hunk 5]
```diff
@@ -111,9 +111,7 @@ NewRelic.setJSAppVersion(version);
 AppRegistry.registerComponent(name, () => App);
 ```
 
-<Callout variant="important">
-Please be sure to enter your proper application tokens. `appToken` is platform-specific. You will need to generate a separate token for Android and iOS apps.
-</Callout>
+Make sure you pasted your application token(s) into `appToken = ""` in the code above. If you deployed your hybrid app to both iOS and Android, you'll need to add two separate tokens, one for iOS and one for Android.
```
Comment (jeff-colucci) [inline src/content/docs/mobile-monitoring/new-relic-monitoring-react-native/monitor-your-react-native-application.mdx:114]: Colon once more

**Review by jeff-colucci — DISMISSED** [review]
Made some comments on some punctuation and a couple of fixes but I think overall it's good and you're good to publish!

File: src/content/docs/mobile-monitoring/new-relic-mobile-cordova-phonegap/get-started-with-cordova-monitoring.mdx [hunk 6]
```diff
@@ -106,6 +106,8 @@ If you're using an [ionic framework](https://ionicframework.com/docs/v1/guide/pr
 
   ```
 
+Make sure you pasted your application token(s) into `appToken = ""` in the code above. If you deployed your hybrid app to both iOS and Android, you'll need to add two separate tokens, one for iOS and one for Android.
```
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile-cordova-phonegap/get-started-with-cordova-monitoring.mdx:109]:
```suggestion
Make sure you paste your application token(s) into `appToken = ""` in the code above. If you deployed your hybrid app to both iOS and Android platforms, you'll need to add two separate tokens: one for iOS and one for Android.
```

File: src/content/docs/mobile-monitoring/new-relic-mobile-flutter/monitor-your-flutter-application.mdx (hunk 1 above)
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile-flutter/monitor-your-flutter-application.mdx:113]:
```suggestion
Make sure you paste your application token(s) into `appToken = ""` in the code above. If you deployed your hybrid app to both iOS and Android platforms, you'll need to add two separate tokens: one for iOS and one for Android.
```

File: src/content/docs/mobile-monitoring/new-relic-mobile-ionic-capacitor/get-started/introduction-new-relic-ionic-capacitor.mdx [hunk 7]
```diff
@@ -111,7 +111,7 @@ let agentConfig : AgentConfiguration = {
 NewRelicCapacitorPlugin.start({appKey:appToken, agentConfiguration:agentConfig})
 ```
 
-2. Replace `appToken` with your application token(s) from Step 1.
+Make sure you pasted your application token(s) into `appToken = ""` in the code above. If you deployed your hybrid app to both iOS and Android, you'll need to add two separate tokens, one for iOS and one for Android.
```
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile-ionic-capacitor/get-started/introduction-new-relic-ionic-capacitor.mdx:114]:
```suggestion
Make sure you paste your application token(s) into `appToken = ""` in the code above. If you deployed your hybrid app to both iOS and Android platforms, you'll need to add two separate tokens: one for iOS and one for Android.
```

File: src/content/docs/mobile-monitoring/new-relic-mobile-maui-dotnet/monitor-your-net-maui-application.mdx (hunk 2 above)
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile-maui-dotnet/monitor-your-net-maui-application.mdx:94]:
```suggestion
Make sure you paste your application token(s) into `appToken = ""` in the code above. If you deployed your hybrid app to both iOS and Android platforms, you'll need to add two separate tokens: one for iOS and one for Android.
```

File: src/content/docs/mobile-monitoring/new-relic-mobile-xamarin/monitor-your-xamarin-application.mdx (hunk 3 above)
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile-xamarin/monitor-your-xamarin-application.mdx:97]:
```suggestion
Make sure you paste your application token(s) into `appToken = ""` in the code above. If you deployed your hybrid app to both iOS and Android platforms, you'll need to add two separate tokens: one for iOS and one for Android.
```

File: src/content/docs/mobile-monitoring/new-relic-monitoring-react-native/monitor-your-react-native-application.mdx (hunk 5 above)
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/new-relic-monitoring-react-native/monitor-your-react-native-application.mdx:114]:
```suggestion
Make sure you paste your application token(s) into `appToken = ""` in the code above. If you deployed your hybrid app to both iOS and Android platforms, you'll need to add two separate tokens: one for iOS and one for Android.
```

**Comment by ally-sassman** [comment]
Got approval from Jeff, merging after pushing up a new commit

**Pull request merged**